app/main.py

```diff
@@ -1,4 +1,8 @@
 def get_human_age(cat_age: int, dog_age: int) -> list:
+    if not is_valid_age(cat_age) or not is_valid_age(dog_age):
+        raise ValueError
```

Comment:
The check for valid age should come after checking the type of the input. Otherwise, a non-integer input will cause an error in `is_valid_age` function.